.doc_gen/templates/zonbook/sdk_chapter_template.xml

```diff
@@ -8,103 +8,151 @@
+            <listitem><para>Sample applications that combine multiple &AWS; services.</para></listitem>
+        </varlistentry>
+    </variablelist>
+    <para role="topiclist-abbrev">Examples</para>
     {{- if .ApiExamples}}
     <section id="{{$.LanguageSlug}}_code_examples_categorized" role="topic">
         <info>
-            <title id="{{$.LanguageSlug}}_code_examples_categorized.title">Examples using {{$.SdkEntity.Short}}</title>
-            <titleabbrev id="{{$.LanguageSlug}}_code_examples_categorized.titleabbrev">Examples</titleabbrev>
+            <title id="{{$.LanguageSlug}}_code_examples_categorized.title">Single-service examples using {{$.SdkEntity.Short}}</title>
+            <titleabbrev id="{{$.LanguageSlug}}_code_examples_categorized.titleabbrev">Single-service</titleabbrev>
             <abstract>
-                <para>The following code examples show how to use {{$.SdkEntity.Short}} with &AWS; services.</para>
+                <para>The following code examples show how to implement common scenarios and perform individual service actions by using
+                    {{$.SdkEntity.Short}} with &AWS; services.</para>
             </abstract>
         </info>
-        <para>The following code examples show how to use {{$.SdkEntity.Short}} with &AWS; services.</para>
-        <para role="topiclist-abbrev">Examples</para>
+        <para>The following code examples show how to perform actions and implement common scenarios by using
+            {{$.SdkEntity.Short}} with &AWS; services.
+            <emphasis>Actions</emphasis> are excerpts that illustrate how to write code to call individual service functions.
+            <emphasis>Scenarios</emphasis> demonstrate ways you can write code to accomplish a specific task with a single service.</para>
+        <para role="topiclist-abbrev">Services</para>
         {{- end}}
         {{- range $service, $svc_examples := .ApiExamples}}
         <section id="{{$.LanguageSlug}}_{{$svc_examples.ServiceSlug}}_code_examples" role="topic">
             <info>
                 <title id="{{$.LanguageSlug}}_{{$svc_examples.ServiceSlug}}_code_examples.title">{{$svc_examples.ServiceEntity.Short}} examples using {{$.SdkEntity.Short}}</title>
-                <titleabbrev id="{{$.LanguageSlug}}_{{$svc_examples.ServiceSlug}}_code_examples.titleabbrev">{{$svc_examples.ServiceEntity.Short}} examples</titleabbrev>
+                <titleabbrev id="{{$.LanguageSlug}}_{{$svc_examples.ServiceSlug}}_code_examples.titleabbrev">{{$svc_examples.ServiceEntity.Short}}</titleabbrev>
                 <abstract>
                     <para>Code examples that show how to use {{$.SdkEntity.Long}} with {{$svc_examples.ServiceEntity.Short}}.</para>
                 </abstract>
             </info>
+            <para>The following code examples show how to perform actions and implement common scenarios by using
+                {{$.SdkEntity.Short}} with {{$svc_examples.ServiceEntity.Short}}.
```

Comment:
the {{$.SdkEntity.Long}} with {{$svc_examples.ServiceEntity.Long}}.

Reply:
Done